denops/aider/buffer.ts

```
@@ -28,207 +28,211 @@ export type BufferLayout = typeof bufferLayouts[number];
+ * If openBufferType is "split" or "vsplit", the buffer is opened in a split.
+ * Otherwise, the buffer is opened in a floating window.
+ *
+ * @param {Denops} denops - The Denops instance.
+ * @param {BufferLayout} openBufferType - The type of buffer to open.
+ * @returns {Promise<void | undefined | boolean>}
+ * @throws {Error} If openBufferType is an invalid value.
+ */
+export async function openAiderBuffer(
+  denops: Denops,
+  openBufferType: BufferLayout,
+): Promise<void | undefined | boolean> {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)